### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by shang624cui** [maszhongming/Effective_Extractive_Summarization#1]
Hi,
I want to know the format of data, for I cannot download it from the link given.
Thanks

**Comment by maszhongming** [comment]
Each article is a JSON file with five keys: "id", "article", "abstract", "label" and "source".
For example, 2222.json in test set is:
{
    "id": "bdc1da890592c918a3ed56db103a18f52e0c1f0a",
    "article": [
        "jordon ibe 's proved he 's got some impressive moves on the football pitch but the young winger has shown us he 's an equally adept dancer - by pulling off the trademark michael jackson moonwalk . '",
        "the liverpool star has broken into the first team set-up this season , with some eye-catching performances at wing back or in a more attacking role .",
        "and ibe took to instagram , to show off his ability to perform the dance move , that many people around the country wish they could replicate .",
        "jordon ibe posted on instagram , the video of him performing the moonwalk while he recovers from injury .",
        "the midfielder showed off some impressive moves as he steps up his recovery from a knee problem .",
        "he said : ` having a little mess around , back training in a few days ... #ooohkillem #mj . '",
        "the 19-year-old is currently out with a knee injury but is expected to return in the next few weeks and boost brendan rodgers ' side , as they continue to chase the champions league places .",
        "liverpool face a struggling newcastle in their next premier league outing as they look to close the gap on fourth .",
        "after being thumped by arsenal at the emirates , the merseysiders know they ca n't afford to slip up in any of their remaining games .",
        "the 19-year-old has done well since breaking into the liverpool first team and is seen as a key member ."
    ],
    "abstract": [
        "jordan ibe showed off the impressive dance move on his instagram .",
        "the liverpool star has broken into the first team during this campaign .",
        "ibe is currently on the sidelines after suffering a knee injury .",
        "click here for all the latest liverpool news ."
    ],
    "label": [
            1,
            4,
            3
    ],
    "source": "DailyMail"
}
However, I still recommend that you should try to download the dataset because as the README.md says, it also contains some pre-trained word2vec files.
